packages/std/schema/init_result.json

```diff
@@ -1,25 +1,25 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "InitResult",
-  "anyOf": [
+  "oneOf": [
```

Comment:
One of seems nicer for enums. Where was this defined?  I think it is better if all the json schema of enums use `oneOf` for clarity

Reply:
For Result: https://github.com/GREsau/schemars/blob/v0.7.1/schemars/src/json_schema_impls/core.rs#L106

For other enums, you see `any_of` here: https://github.com/GREsau/schemars/blob/63af0ceb73e177bc792a7b256b223c3072910313/schemars_derive/src/lib.rs#L128-L260 But I don't understand why this was chosen.

Reply:
Hmmm.. I see those `wrap_schema_fields` calls, but don't really understand.
If you want to raise an issue on schemars, I'd be curious (or I can later). But definitely not a blocker.